kyuubi-server/src/test/scala/org/apache/kyuubi/engine/EngineRefSuite.scala

```diff
@@ -205,6 +208,8 @@ class EngineRefSuite extends KyuubiFunSuite {
     conf.set(ENGINE_POOL_SIZE, 3)
     val engine6 = new EngineRef(conf, user, id, null)
     assert(engine6.subdomain.startsWith(s"$enginePoolName-"))
+    conf.unset(ENGINE_POOL_NAME)
+    conf.unset(ENGINE_POOL_SIZE)
```

Comment:
seems not needed because you have added it into `beforeEach`

Reply:
yeah, let me remove this first to see any issue in test